src/oas3/transformers/content.ts

```diff
@@ -85,7 +86,11 @@ export function translateMediaTypeObject(
   return {
     mediaType,
     schema: schema
-      ? (toJsonSchema(schema, { cloneSchema: true, keepNotSupported: ['example'] }) as JSONSchema4)
+      ? (toJsonSchema(schema, {
+          cloneSchema: true,
+          strictMode: false,
+          keepNotSupported: ['example'],
+        }) as JSONSchema4)
```

Comment:
Has it got a JSONSchema5 tucked away in there? They should be the same but OAS2=JS4 and OAS3=JS5 so if we can use the right interface that'd potentially remove a bit of confusion somewhere.

Reply:
I don’t know why that cast was added there, it was already there when I made the changes. What does that open api library actually return?